### ISSUE TITLE
Api stopped to return an html for a twitch live

### ISSUE THREAD

**Issue by AndrzejSala** [itteco/iframely#194]
Hi guys!
I have just noticed that the Iframely API stopped to return an html property for twitch live embeds despite that http://open.iframe.ly returns it correctly. I'm sure that some weeks ago the html prop was inside a return, because I was using it in my application and everything was correct.

Example url: **https://www.twitch.tv/ninja**

**http://open.iframe.ly**
```
{
	"url": "https://www.twitch.tv/ninja",
	"type": "video",
	"version": "1.0",
	"title": "Ninja - Twitch",
	"provider_name": "Twitch",
	"description": "Tournament Rerun *Not Live*",
	"thumbnail_url": "https://static-cdn.jtvnw.net/jtv_user_pictures/6d942669-203f-464d-8623-db376ff971e0-profile_image-300x300.png",
	"thumbnail_width": 300,
	"thumbnail_height": 300,
	"html": "<div style=\"left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.2493%;\"><iframe src=\"//player.twitch.tv/?channel=ninja&amp;player=facebook&amp;autoplay=false\" style=\"border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;\" allowfullscreen></iframe></div>",
	"cache_age": 86400
}
```

**Self-hosted iframely**
```
{
	"meta": {
		"title": "Twitch",
		"canonical": "https://www.twitch.tv/ninja",
		"site": "Twitch",
		"description": "Twitch is the world's leading video platform and community for gamers."
	},
	"links": {
		"thumbnail": [{
			"href": "//www-cdn.jtvnw.net/images/twitch_logo3.jpg",
			"type": "image/jpeg",
			"rel": ["thumbnail", "og", "ssl"]
		}],
		"icon": [{
			"href": "https://static.twitchcdn.net/assets/favicon-75270f9df2b07174c23c.ico",
			"rel": ["icon", "ssl"],
			"type": "image/icon"
		}]
	},
	"rel": []
}
```

The return from normal twitch videos like https://www.twitch.tv/videos/287127728 contain html prop inside, the problem is only with lives.

**Comment by iparamonau** [comment]
Twitch was changing things couple months ago, but Iframely version 1.2.5 has the same plugin we have in the cloud. What version do you use?

**Comment by AndrzejSala** [comment]
@iparamonau 
I was using the newest - 1.2.5

**Comment by iparamonau** [comment]
@adamhooper Actually, I was mistaken - we did do a fix after v1.2.5. We'll try and cut a new version tomorrow. In the meantime, you may try to pull and replace this file from `develop` branch as a patch:
https://github.com/itteco/iframely/blob/develop/plugins/domains/twitch.tv.js

**Comment by iparamonau** [comment]
Should now be resolved in v 1.2.6
